Mathlib/GroupTheory/GroupAction/FixedPoints.lean

```diff
@@ -0,0 +1,286 @@
+/-
+Copyright (c) 2024 Emilie Burgun. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Emilie Burgun
+-/
+import Mathlib.GroupTheory.GroupAction.Basic
+import Mathlib.Dynamics.PeriodicPts
+import Mathlib.Data.Set.Pointwise.SMul
+
+/-!
+# Properties of `fixedPoints` and `fixedBy`
+
+This module contains some useful properties of [`MulAction.fixedPoints`] and [`MulAction.fixedBy`]
+that don't directly belong to `Mathlib.GroupTheory.GroupAction.Basic`,
+as well as the definition of [`MulAction.movedBy`].
+
+## Main theorems
+
+* [`MulAction.fixedBy_mul`], [`MulAction.movedBy_mul`] (and their `AddAction` equivalents),
+  for the different relationship between the `movedBy`/`fixedBy` sets of `g*h`.
+* [`MulAction.fixedBy_conj`] and [`MulAction.movedBy_conj`]: the pointwise group action on the sets
+  `fixedBy α g` and `movedBy α g` translates to the conjugation action on `g`.
+* [`MulAction.smul_in_set_of_movedBy_subset`] shows that if a set `s` is a superset of
+  `movedBy α g`, then the group action of `g` cannot send elements of `s` outside of `s`.
+* [`MulAction.not_commute_of_disjoint_movedBy_preimage`] allows one to infer that two group elements
+  do not commute from the behavior of their `movedBy` sets, which is useful in the proof of
+  Rubin's theorem.
+-/
+
+namespace MulAction
+open Pointwise
+
+universe u v
+variable {α : Type v}
+variable {G : Type u} [Group G] [MulAction G α]
+variable {M : Type u} [Monoid M] [MulAction M α]
+
+
+section FixedPoints
+
+/-- In a multiplicative group action, the points fixed by `g` are also fixed by `g⁻¹` -/
+@[to_additive "In an additive group action, the points fixed by `g` are also fixed by `g⁻¹`"]
+theorem fixedBy_eq_fixedBy_inv (g : G) : fixedBy α g = fixedBy α g⁻¹ := by
```

Comment:
Should `α` be explicit here?

Reply:
It should, yes, although someone should probably do an automated pass on the entirety of mathlib because I've had to manually force-provide arguments several times